### PR TITLE
[bitnami/minio] Update README.md - auth.existingSecret error

### PR DESCRIPTION
**Pull request by JustinGuese**
The secret for the password is called "root-password" and not mariadb-root-password

**Description of the change**

Only Readme (1 line)

**Benefits**

Showing the real secret

**Possible drawbacks**

None

**Applicable issues**

  - fixes #

**Additional information**

Keep on providing such nice charts!

**Checklist** 

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [X ] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/minio/README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ The command removes all the Kubernetes components associated with the chart and
...
 | `auth.rootUser`            | MinIO&reg; root username                                                                                                                                                                                         | `admin`                  |
 | `auth.rootPassword`        | Password for MinIO&reg; root user                                                                                                                                                                                | `""`                     |
-| `auth.existingSecret`      | Use existing secret for credentials details (`auth.rootUser` and `auth.rootPassword` will be ignored and picked up from this secret). The secret has to contain the keys `root-user` and `mariadb-root-password) | `""`                     |
+| `auth.existingSecret`      | Use existing secret for credentials details (`auth.rootUser` and `auth.rootPassword` will be ignored and picked up from this secret). The secret has to contain the keys `root-user` and `root-password`) | `""`                     |
```
Comment (juan131) [inline bitnami/minio/README.md:90]: The README table is autogenerated based on the metadata available in the **values.yaml** using the [eadme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool. Therefore, you need to fix this in the values.yaml (check the line below) and run the tool to update the table:

- https://github.com/bitnami/charts/blob/master/bitnami/minio/values.yaml#L97

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for reporting this issue!! Please check my comments

File: bitnami/minio/README.md (hunk 1 above)
Comment (JustinGuese) [reply bitnami/minio/README.md:90]: Alright, thanks @juan131 . Could you point to the tool to generate the readme? I couldn't find any...

Comment (juan131) [reply bitnami/minio/README.md:90]: I added a link in my comment, here you have it in any case: https://github.com/bitnami-labs/readme-generator-for-helm

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**